src/pages/home/report/ReportActionsView.js

```diff
@@ -154,11 +159,36 @@ function ReportActionsView(props) {
         if (oldestReportAction.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED) {
             return;
         }
-
         // Retrieve the next REPORT.ACTIONS.LIMIT sized page of comments
-        Report.readOldestAction(reportID, oldestReportAction.reportActionID);
+        Report.getOlderActions(reportID, oldestReportAction.reportActionID);
     };
 
+    /**
+     * Retrieves the next set of report actions for the chat once we are nearing the end of what we are currently
+     * displaying.
+     */
+    const loadNewerChats = _.throttle(({distanceFromStart}) => {
+        // Only fetch more if we are not already fetching so that we don't initiate duplicate requests.
+        if (props.isLoadingNewerReportActions || props.isLoadingInitialReportActions) {
+            return;
+        }
+
+        // Ideally, we wouldn't need to use the 'distanceFromStart' variable. However, due to the low value set for 'maxToRenderPerBatch',
+        // the component undergoes frequent re-renders. This frequent re-rendering triggers the 'onStartReached' callback multiple times.
+        //
+        // To mitigate this issue, we use 'CONST.CHAT_HEADER_LOADER_HEIGHT' as a threshold. This ensures that 'onStartReached' is not
+        // triggered unnecessarily when the chat is initially opened or when the user has reached the end of the list but hasn't scrolled further.
+        //
+        // Additionally, we use throttling on the 'onStartReached' callback to further reduce the frequency of its invocation.
+        // This should be removed once the issue of frequent re-renders is resolved.
+        if (!isFetchNewerWasCalled.current || distanceFromStart <= CONST.CHAT_HEADER_LOADER_HEIGHT) {
+            isFetchNewerWasCalled.current = true;
+            return;
+        }
```

Comment:
`onStartReached` is triggered during the first render. Since we use `OpenReport` on the first render and are confident about the message ordering, we can safely skip this call

Reply:
Can you please rename the ref to isFirstRender and add a comment for that

Reply:
👍🏻